locale/fi/LC_MESSAGES/django.po

```diff
@@ -747,6 +747,46 @@ msgstr "myöhästymissakko"
 msgid "MODULE_LATE_SUBMISSION_PENALTY_HELPTEXT"
 msgstr "Pisteiden vähennyskerroin desimaalilukuna, esim. 0,1 = 10%"
 
+#: exercise/templates/exercise/_user_results.html
+msgid "POINTS_GOAL"
+msgstr "Pistetavoite"
+
+#: exercise/templates/exercise/_user_results.html
+msgid "POINTS_GOAL_TOOLTIP"
+msgstr "Aseta henkilökohtainen pistetavoite tälle moduulille. Tavoite näkyy vain sinulle. Kun tavoite on saavutettu, pistepalkki vaihtaa värin siniseksi."
+
+#: exercise/templates/exercise/personalized_points_goal_modal.html
+msgid "PERSONALIZED_POINTS_MODAL_TITLE"
+msgstr "Aseta pistetavoite moduulille"
+
+#: exercise/templates/exercise/personalized_points_goal_modal.html
+msgid "LABEL_POINTS_GOAL_INPUT"
+msgstr "Anna pistetavoite prosentteina tai pisteinä (esim. 50% tai 150)"
+
+#: exercise/templates/exercise/personalized_points_goal_modal.html
+msgid "PERSONALIZED_POINTS_MODAL_SUCCESS"
+msgstr "Pistetavoite asetettu onnistuneesti"
+
+#: exercise/templates/exercise/personalized_points_goal_modal.html
+msgid "PERSONALIZED_POINTS_MODAL_FAILURE"
+msgstr "Pistetavoitteen asettaminen epäonnistui"
+
+#: exercise/templates/exercise/personalized_points_goal_modal.html
+msgid "PERSONALIZED_POINTS_MODAL_NOT_NUMBER"
+msgstr "Pistetavoite tulee antaa joko prosentteina tai pisteinä (esim. 50% or 150)"
+
+msgid "PERSONALIZED_POINTS_GOAL"
+msgstr "Pistetavoite"
+
+msgid "PERSONALIZED_POINTS_MODAL_LESS_THAN_REQUIRED"
+msgstr "Pistetavoite ei voi olla vähemmän kuin moduulin läpipääsyyn vaaditut pisteet"
```

Comment:
Could you change `läpipääsyyn` --> `läpäisyyn` so that the wording matches other similar messages.